.github/workflows/buildcheck.yml

```diff
@@ -12,6 +12,7 @@ on:
       - 'lib/**'
       - 'lib.src/**'
       - 'src/**'
+  push:
```

Comment:
Das kann von mir aus weg, aber ich brauchte es, um einfach in meinem Projekt zu bauen

Reply:
> Das kann von mir aus weg, aber ich brauchte es, um einfach in meinem Projekt zu bauen

Kannst du näher erklären, warum du das brauchtest?

Reply:
Der build in meinem fork läuft nur, wenn mir selbst einen pr gestellt hätte. So kann ich einfach pushen und es läuft direkt los. Insgesamt ist ein build check etwas, was aus meiner Sicht immer laufen sollte.